### PR TITLE
cleanup: fix varnamelen linter

### PR DESCRIPTION
**Pull request by riya-singhal31**
This PR fix the varnamelen linter errors caused by golangci-lint 1.43.

For reference- https://github.com/ceph/ceph-csi/issues/2595

Signed-off-by: riya-singhal31 <rsinghal@redhat.com>

### CODE REVIEW

**Comment by nixpanic** [comment]
Still a few issues reported by golangci-lint, you'll probably want to address those as well.

The prefix `fix:` is not commonly used for the patch subject, better use `cleanup: fix varnamelen linter` instead.

**Comment by riya-singhal31** [comment]
> Still a few issues reported by golangci-lint, you'll probably want to address those as well.
> 
> The prefix `fix:` is not commonly used for the patch subject, better use `cleanup: fix varnamelen linter` instead.

Thanks for the review Niels, updated the PR.

File: scripts/golangci.yml.in [hunk 1]
```diff
@@ -156,7 +156,71 @@ linters-settings:
     max-blank-identifiers: 3
   nestif:
     min-complexity: 7
-
+  varnamelen:
+    ignore-names:
```
Comment (nixpanic) [inline scripts/golangci.yml.in:160]: this list gets quite long, I am not sure that is properly addressing the linter concerns... Is there a plan to reduce this list?

**Review by Madhu-1 — COMMENTED** [review]
I see no value in fixing this one and we should leave this as it is, @riya-singhal31 WDYT?

**Comment by riya-singhal31** [comment]
> I see no value in fixing this one and we should leave this as it is, @riya-singhal31 WDYT?

Yes right @Madhu-1 , because to all the variables where name length is < 3, its popping this linter. So doesn't put much difference.

**Pull request closed without merging**